content/knowledge/scalinghistory.md

```diff
@@ -0,0 +1,68 @@
+   However sharding has its own challenges. Corrupting nodes in a given shard may lead to permanent loss of data. One way to tackle this issue is by randomly assigning a node to a shard and randomly reassigning the node to another shard.
+   Sharding has another problem that the composability aspect of Smart contract compatible blockchain will no longer exist. For example, if a Defi built on top of Ethereum1.0 ecosystem relies on or combines few other projects, then in a sharded system of Ethereum2.0, ideally they should be on the same shard which may not always be the case. If they are not on the same shard then they all needs to be ok with asynchronous updates which is [not ideal](https://www.coindesk.com/tech/2020/10/13/will-a-sharded-ethereum-be-flexible-enough-for-decentralized-finance/ ).
+
+## 4. Avalanche
+
+   The next level solution for scalability in terms of consensus algorithms is by Avalanche network. In Avalanche, the underlying [sybil control mechanism](https://en.wikipedia.org/wiki/Sybil_attack) is still PoS but there is a different mechanism in place to finalise validating transactions. This is achieved by a particular node randomly subsampling from other nodes multiple times across multiple cycles and eventually arriving at a consensus. This is then recorded in a Directed Acyclic Graph as opposed to a chain of Blocks. This consensus can easily scale at a much higher level compared to Bitcoin/PoW or Ethereum2.0/PoS because of faster agreement of transactions.
+   The issue with Avalanche is similar to what other PoS based networks also have i.e. the economic limit of having a lot of value locked as stake in order to have the PoS running which may not be practical when the network overall value is in trillions of dollars. The minimum staking that is required to become a validator may not be easy for an average person to be a validator without delegating to another validator thereby contributing to centralisation of the network. Thus Avalanche has validators only in the number of thousands and not tens or hundreds of thousands so far.
+   Usually many PoS based networks impose a penalty for malicious validators or validators that aren’t online continuously by slashing which is taking away some of the staked tokens as a punishment. Avalanche doesn’t have this mechanism to encourage more participation as a step towards more decentralisation thereby sacrificing a bit more of security.
+
+   (Doubt: How the rich get richer compounding effect is absent in Avalanche?)
```

Comment:
How the rich get richer compounding effect is absent in Avalanche? Since bigger validators will have higher chance of reward?

Reply:
I don't think that was ever a stated goal of any project. Why do you bring it up?

It's definitely something worth talking about, and many people have, but it's not within the scope of this document.